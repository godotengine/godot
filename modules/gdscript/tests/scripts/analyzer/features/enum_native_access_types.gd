func print_enum(e: TileSet.TileShape) -> TileSet.TileShape:
	print(e)
	return e

func test():
	var v: TileSet.TileShape
	v = TileSet.TILE_SHAPE_SQUARE
	v = print_enum(v)
	v = print_enum(TileSet.TILE_SHAPE_SQUARE)
	v = TileSet.TileShape.TILE_SHAPE_SQUARE
	v = print_enum(v)
	v = print_enum(TileSet.TileShape.TILE_SHAPE_SQUARE)

	v = TileSet.TILE_SHAPE_ISOMETRIC
	v = print_enum(v)
	v = print_enum(TileSet.TILE_SHAPE_ISOMETRIC)
	v = TileSet.TileShape.TILE_SHAPE_ISOMETRIC
	v = print_enum(v)
	v = print_enum(TileSet.TileShape.TILE_SHAPE_ISOMETRIC)
