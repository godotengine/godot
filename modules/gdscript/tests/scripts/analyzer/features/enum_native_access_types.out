GDTEST_OK
0
0
0
0
1
1
1
1
